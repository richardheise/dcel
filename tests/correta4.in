4 2
0 0
2 0
2 2
0 2
1 2 4
2 3 4
3 2 1 4
