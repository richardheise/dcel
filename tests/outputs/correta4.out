4 5 3
0 0 1
2 0 2
2 2 5
0 2 3
1
4
7
1 8 1 2 3
2 6 1 3 1
4 9 1 1 2
2 7 2 5 6
3 10 2 6 4
4 2 2 4 5
3 4 3 8 10
2 1 3 9 7
1 3 3 10 8
4 5 3 7 9
